common/logger/src/macros.rs

```diff
@@ -99,6 +99,34 @@ macro_rules! schema {
         )
     };
 
+    // Identifier Keys debug
+    (@ { $(,)* $($out:expr),* }, ?$($k:ident).+ = $val:expr, $($args:tt)*) => {
+        $crate::schema!(
+            @ { $($out),*, &$crate::KeyValue::new($crate::__log_stringify!($($k).+), $crate::Value::from_serde(&format!("{:?}", $val))) },
```

Comment:
```suggestion
            @ { $($out),*, &$crate::KeyValue::new($crate::__log_stringify!($($k).+), $crate::Value::from_debug(&$val)) },
```

Don't allocate, instead use the from_debug/from_display functions on the Value type

Reply:
sound good.